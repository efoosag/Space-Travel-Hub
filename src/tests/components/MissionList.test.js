import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import MissionList from '../../components/MissionList';

describe('Testing MissionList component', () => {
  it('Testing with snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <MissionList />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
