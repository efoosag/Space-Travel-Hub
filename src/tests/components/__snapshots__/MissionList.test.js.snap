// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing MissionList component Testing with snapshot 1`] = `
<section
  className="section"
>
  <table
    className="mission-table"
  >
    <thead>
      <th
        className="width-10"
      >
        Mission
      </th>
      <th
        className="width-60"
      >
        Description
      </th>
      <th
        className="width-15"
      >
        Status
      </th>
      <th
        className="width-15"
      >
        Action
      </th>
    </thead>
    <tbody />
  </table>
</section>
`;
